.github/workflows/build_validation_develop.yml

```diff
@@ -52,6 +52,11 @@ jobs:
             terraform_workspace_services:
               - templates/workspace_services/**/terraform/**/*.tf
 
+      - uses: hashicorp/setup-terraform@v3
+        if: ${{ steps.filter.outputs.terraform == 'true' }}
+        with:
+          terraform_version: "1.9.8"
```

Comment:
Does this need to be done across all workflows, and should the version be a variable?

Reply:
There's only 2 workflows that uses Terraform directly. Lets encrypt one already uses this process.
Not sure if we need a var here

Reply:
Ok, as its in the dev container for the others and this doesn't use the dev container? Is that the difference?

Reply:
Yes. This is a quick lint validation so we don't need to wait for the devcontainer or superlinter for that matter.